### DISCUSSION TITLE
create bootstrap : how to clean leftovers : how to use dumpunreferenced/deleteunreferenced

### DISCUSSION THREAD

**Discussion by ipanmeri** [uyuni-project/uyuni#8639] [Q&A] [answered]
Hi,
while trying to

`mgr-create-bootstrap-repo -c ubuntu-20.04-amd64-uyuni`

at the end I get this output:

`Skipping inclusion of 'libzmq5' '4.3.2-2ubuntu1' in 'bootstrap|main|amd64', as this version already exists.
Deleting files just added to the pool but not used.
(to avoid use --keepunusednewfiles next time)
Not deleting possibly left over files due to previous errors.
(To keep the files in the still existing index files from vanishing)
Use dumpunreferenced/deleteunreferenced to show/delete files without references.
Warning: database 'bootstrap|main|amd64' was modified but no index file was exported.
Changes will only be visible after the next 'export'!
Warning: database 'bootstrap|main|i386' was modified but no index file was exported.
Changes will only be visible after the next 'export'!
Warning: database 'bootstrap|main|armhf' was modified but no index file was exported.
Changes will only be visible after the next 'export'!
Warning: database 'bootstrap|main|arm64' was modified but no index file was exported.
Changes will only be visible after the next 'export'!
There have been errors!
Error creating bootstrap repo.
You have new mail in /var/spool/mail/root
`

What are the "dumpunreferenced/deleteunreferenced" options and how do I apply them?
Command reference does not explain.

`Usage: mgr-create-bootstrap-repo [options] [additional_pkg1 additional_pkg2 ...]

Tool to generate repositories containing the required software to register at
SUSE Manager Server. Logs are written to /var/log/rhn/mgr-create-bootstrap-
repo/mgr-create-bootstrap-repo.log .

Options:
  -h, --help            show this help message and exit
  -n, --dryrun          Dry run. Show only changes - do not execute them
  -i, --interactive     Interactive mode (default)
  -l, --list            list available distributions
  -c CREATE, --create=CREATE
                        create bootstrap repo for given distribution label
  -a, --auto            Automatic Mode. Generate all available bootstrap repos
  --datamodule=DATAMODULE
                        Use an own datamodule (Default: mgr_bootstrap_data)
  -d, --debug           Enable debug mode
  -f, --flush           when used in conjuction with --create, deletes the
                        target repository before creating it (default)
  --no-flush            when used in conjuction with --create, prevent
                        deletion of the target repository before creating it
  --force               Force creation even when not all required channels are
                        available
  --with-custom-channels
                        Take custom channels into account when searching for
                        newest package versions
  --with-parent-channel=PARENTCHANNEL
                        use child channels below this parent
`

Or, how do I "reset" a bootstrap repo?
Thanks

**Comment by rjmateus** [comment, marked as answer]
bootstrap repositories are saved at `/srv/www/htdocs/pub/repositories`. You can try to find the problematic one and just delete the folder

**Reply by ipanmeri** [reply]
Thanks
